include/fiction/layouts/coordinates.hpp

```diff
@@ -665,6 +669,21 @@ struct formatter<fiction::cube::coord_t>
 };
 }  // namespace fmt
 
-#pragma GCC diagnostic pop
+namespace siqad
+{
+using coord_t = fiction::cube::coord_t;
+template <typename CoordinateType>
+CoordinateType constexpr siqad_to_other(const coord_t& coord) noexcept
+{
+    return {coord.x, coord.y*2 + coord.z};
+};
 
+template <typename CoordinateType>
+coord_t constexpr other_to_siqad(const CoordinateType& coord) noexcept
```

Comment:
I would rename the two functions to something a little more expressive like `to_siqad_coord` and `to_fiction_coord`